wagtail_transfer/views.py

```diff
@@ -97,8 +100,29 @@ def objects_for_export(request):
     }, json_dumps_params={'indent': 2})
 
 
+class UIDField(Field):
+    """
+    Serializes UID for the Page Chooser API
+    """
+    def get_attribute(self, instance):
+        return instance
+
+    def to_representation(self, page):
+        return get_locator_for_model(Page).get_uid_for_local_id(page.id, create=False)
```

Comment:
I think this should be in `get_attribute`?

Reply:
That makes sense - is there a reason the PagesAPI doesn't do that then? Fixed now anyway

Reply:
Ahh, might be that I wrote that code a long time ago!